wordpress-mobile/WordPress-iOS
# Remove api client property from the blog object.

Refs #5245 
This PR removes the old ```api``` XMLRPCClient property from the blog object. The property was replaced by the ```xmlrpcApi``` object so it's no longer needed.

To test:
 - Run the app and add a self-hosted site.
 - Check if the main section for the blog work correctly: blog, comments, posts, pages, settings

Needs review: @diegoreymendez
